setup.py

```diff
@@ -14,6 +15,12 @@
     .strip().strip('\'"')
 )
 
+install_requires = [
+    'ifaddr'
+]
+if sys.version_info[:2] < (3, 5):
+    install_requires.append('typing')
```

Comment:
This makes it so that binary packages created on Python 3.5+ won't have the typing dependency which is not what we want. 
https://github.com/eventlet/eventlet/commit/b4b20c459af34d54b52e98a72fe68f6e5143398f shows the way to do it.

Reply:
Isn't the `typing` module included in 3.5+? And if I understand correctly, binary wheels are specific to python versions, so you would be producing them independently for each of them, and therefore the dependency should appear correctly.

The difference then would only on how the dependency is stated and processed (I think I'm still stuck to the old ways of setuptools); I'll change it now.

Reply:
Yes, it's included in Python 3.5. The correct requirement scoping will only install the extra dependency on Python 3.4.

We provide universal wheels (see https://github.com/jstasiak/python-zeroconf/blob/b03cee348973469e9ebfce6e9b0e0a367c146401/setup.cfg#L2)

Reply:
Ohh, I didn't realize that, sorry, something new I learned today...